sdk/synapse/Azure.Analytics.Synapse.Artifacts/tests/samples/NotebookSampleSnippets.cs

```diff
@@ -43,36 +33,24 @@ public void CreateNotebook()
                 new List<NotebookCell>()
             );
             string notebookName = "MyNotebook";
-            NotebookCreateOrUpdateNotebookOperation operation = notebookClient.StartCreateOrUpdateNotebook(notebookName, new NotebookResource(notebookName, notebook));
+            NotebookCreateOrUpdateNotebookOperation operation = client.StartCreateOrUpdateNotebook(notebookName, new NotebookResource(notebookName, newNotebook));
             NotebookResource notebookResource = operation.WaitForCompletionAsync().ConfigureAwait(true).GetAwaiter().GetResult();
             #endregion
-        }
 
-        [Test]
-        public void RetrieveNotebook()
-        {
             #region Snippet:RetrieveNotebook
-            NotebookResource notebook = notebookClient.GetNotebook("MyNotebook");
+            NotebookResource notebook = client.GetNotebook("MyNotebook");
             #endregion
-        }
 
-        [Test]
-        public void ListNotebooks()
-        {
             #region Snippet:ListNotebooks
-            Pageable<NotebookResource> notebooks = notebookClient.GetNotebooksByWorkspace();
-            foreach (NotebookResource notebook in notebooks)
+            Pageable<NotebookResource> notebooks = client.GetNotebooksByWorkspace();
+            foreach (NotebookResource book in notebooks)
```

Comment:
nit: same as others - I would avoid the abbreviated names.

Reply:
Same as above (naming conflict).